content/applications/hr/payroll/payslips.rst

```diff
@@ -28,159 +30,178 @@ from this dashboard.
    :align: center
    :alt: View all payslips that need to be paid on the Payslips To Pay page.
 
-Each payslip will list the :guilabel:`Reference` number for the individual payslip, the
-:guilabel:`Employee` name, :guilabel:`Batch Name`, the :guilabel:`From` and :guilabel:`To` date
-range, the :guilabel:`Company`, the :guilabel:`Basic Wage`, :guilabel:`Net Wage`, and the
-:guilabel:`Status` of the payslip.
+Each payslip lists the :guilabel:`Reference` number for the individual payslip, the
+:guilabel:`Employee` name, the :guilabel:`Batch Name`, the :guilabel:`Company`, the :guilabel:`Basic
+Wage`, :guilabel:`Gross Wage`, :guilabel:`Net Wage`, and the :guilabel:`Status` of the payslip.
 
-Clicking on an individual payslip entry will show the details for the individual payslip.
+Click on an individual payslip entry to view the details for that individual payslip.
 
 .. _payroll/new-payslip:
 
-Create new payslip
-------------------
+Create a new payslip
+--------------------
 
-A new payslip can be created from the :guilabel:`Payslips To Pay` page (:menuselection:`Payroll app
---> Payslips --> To Pay`) or the :guilabel:`Employee Payslips` page (:menuselection:`Payroll app
---> Payslips --> All Payslips`), by clicking the :guilabel:`Create` button in the top-left corner.
+A new payslip can be created from either the :ref:`Payslips To Pay <payroll/to-pay>` page or the
+:ref:`Employee Payslips <payroll/all-payslips>` page.
 
-Clicking :guilabel:`Create` reveals a blank payslip form, wherein the necessary payslip information
-can be entered.
+Create a new payslip by clicking the :guilabel:`New` button in the top-left corner.
+
+A blank payslip form is loaded, where the necessary payslip information can be entered.
 
 Required fields
 ~~~~~~~~~~~~~~~
 
-On the blank payslip form, a number of fields are required to be filled in with the necessary
-information. These required fields are represented by **bold** lines.
+On the blank payslip form, several fields are required. Most of the required fields auto-populate
+after an employee is selected.
+
+Enter the following information on the new payslip form:
+
+- :guilabel:`Employee`: type in the name of an employee, or select the desired employee from the
+  drop-down list.
+- :guilabel:`Period`: the current month (the first day to the last day) auto-populates the
+  :guilabel:`Period` fields by default. The dates can be changed, if desired.
+
+  To change the start date, click on the first date field to reveal a pop-up calendar. On this
+  calendar, use the :guilabel:`< (less-than)` and :guilabel:`> (greater-than)` icons to select the
+  desired month, then click on the desired day to select that specific date.
+
+  Repeat this process to modify the end date for the payslip.
+- :guilabel:`Contract`: using the drop-down menu, select the desired contract for the employee. Only
+  the available corresponding contracts for the selected employee appear as options.
```

Comment:
I made a note in the Employee part- since people should ONLY be writing pay slips for employees in the system (doesn't make sense to not follow that order- they are hired, added to the system, with a contract, then after they have worked, they get a paycheck. Agree? Made sense to me!) I think the note should clear it up, and it links to the doc explaining how to do all that.